runtime/vm/CRIUHelpers.cpp

```diff
@@ -1750,8 +1750,29 @@ criuCheckpointJVMImpl(JNIEnv *env,
 		VM_VMHelpers::setVMState(currentThread, J9VMSTATE_CRIU_SUPPORT_RESTORE_PHASE_START);
 		restoreNanoTimeMonotonic = j9time_nano_time();
 		restoreNanoUTCTime = j9time_current_time_nanos(&success);
-		vm->checkpointState.lastRestoreTimeMillis = (I_64)(restoreNanoUTCTime / J9TIME_NANOSECONDS_PER_MILLIS);
-		Trc_VM_criu_after_dump(currentThread, restoreNanoTimeMonotonic, restoreNanoUTCTime, vm->checkpointState.lastRestoreTimeMillis);
+		if (0 == success) {
+			systemReturnCode = errno;
+			currentExceptionClass = vm->checkpointState.criuSystemRestoreExceptionClass;
+			nlsMsgFormat = j9nls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
+				J9NLS_VM_CRIU_J9_CURRENT_TIME_NANOS_FAILURE,
+				NULL);
```

Comment:
One argument per line; also line 1768.

Reply:
Would one argument per line here be:

```
nlsMsgFormat = j9nls_lookup_message(
	J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
	J9NLS_VM_CRIU_J9_CURRENT_TIME_NANOS_FAILURE,
	NULL);
```
?

Reply:
Yes, like code you added in `runtime/tests/port/si.c`.